src/lib/Strategy/FocusMode/ContentStructureRedirectStrategy.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Strategy\FocusMode;
+
+use Ibexa\Contracts\AdminUi\FocusMode\RedirectStrategyInterface;
+use Ibexa\Contracts\Core\Repository\LocationService;
+use Ibexa\Contracts\Core\SiteAccess\ConfigResolverInterface;
+use Symfony\Component\Routing\RouterInterface;
+
+final class ContentStructureRedirectStrategy implements RedirectStrategyInterface
+{
+    private ConfigResolverInterface $configResolver;
+
+    private LocationService $locationService;
+
+    private RouterInterface $router;
+
+    public function __construct(
+        ConfigResolverInterface $configResolver,
+        LocationService $locationService,
+        RouterInterface $router
+    ) {
+        $this->configResolver = $configResolver;
+        $this->locationService = $locationService;
+        $this->router = $router;
+    }
+
+    public function supports(string $route): bool
+    {
+        return in_array(
+            $route,
+            [
+                'ibexa.section.list',
+                'ibexa.content_type_group.list',
+                'ibexa.object_state.groups.list',
+                'ibexa.content_type_group.view',
+            ],
```

Comment:
Can this list be acquired from elsewhere, for example in a some menu builder?

Reply:
Actually, that's why I introduced an extension point here - the default behavior can be circumvented. There is no "list" of columns/menu items hidden in focus mode that I can utilize, unfortunately.

Reply:
As discussed, I'd suggest using a route attribute to determine that it is not available in Focus Mode.

Reply:
It seems route attributes/options cannot be easily fetched even though we are able to match specific routes. Decided to stick to the original approach.